apps/server/src/shared/domain/entity/lesson.entity.ts

```diff
@@ -87,6 +89,10 @@ export class Lesson extends BaseEntityWithTimestamps implements ILearnroomElemen
 	@ManyToOne('Course', { fieldName: 'courseId' })
 	course: Course;
 
+	@Index()
```

Comment:
Do we have a query that search lessons by his coursgroups for now, or it is a pro active index?